internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -104,6 +112,7 @@ func DecryptBuilder() *cobra.Command {
 	cmd.Flags().StringVarP(&opts.localKeyFileName, flag.LocalKeyFile, "", "", usage.LocalKeyFile)
 	cmd.Flags().StringVarP(&opts.kmipServerCAFileName, flag.KMIPServerCAFile, "", "", usage.KMIPServerCAFile)
 	cmd.Flags().StringVarP(&opts.kmipClientCertificateFileName, flag.KMIPClientCertificateFile, "", "", usage.KMIPClientCertificateFile)
+	cmd.Flags().StringVarP(&opts.kmipClientCertificatePassword, flag.KMIPClientCertificatePassword, "", "", usage.KMIPClientCertificatePassword)
```

Comment:
this and all other flags that don't take a shorthand
```suggestion
	cmd.Flags().StringVar(&opts.kmipClientCertificatePassword, flag.KMIPClientCertificatePassword, "", usage.KMIPClientCertificatePassword)
```